rpc/block_svc_routines.c

```diff
@@ -3955,9 +3955,8 @@ blockValidateCommandOutput(const char *out, int opt, void *data)
 
     /* target iqn create validation */
     GB_OUT_VALIDATE_OR_GOTO(out, out, "target iqn creation failed for: %s",
-                    cblk, cblk->volume,
-                    "Created target iqn.2016-12.org.gluster-block:%s.",
-                    cblk->gbid);
+                    cblk, cblk->volume, "Created target  %s:%s.",
```

Comment:
You seem to have kept an extra space after "Created target", Can you please remove this addition.
Thanks!